src/ol/renderer/webgl/PointsLayer.js

```diff
@@ -343,45 +399,44 @@ class WebGLPointsLayerRenderer extends WebGLLayerRenderer {
    * @private
    */
   rebuildBuffers_(frameState) {
-    const layer = this.getLayer();
-    const vectorSource = layer.getSource();
-
     // saves the projection transform for the current frame state
     const projectionTransform = createTransform();
     this.helper.makeProjectionTransform(frameState, projectionTransform);
 
-    const features = vectorSource.getFeatures();
+    const featureUids = Object.keys(this.featureCache_);
```

Comment:
Doesn't need to happen here, but since you only want the count of features, we could avoid the array allocation and cache the count when you cache the features.

Reply:
Good find, done in 8aa606170975062dd733e75f19965978bc7e2cbf